code/meta-parent/pom.xml

```diff
@@ -47,7 +47,7 @@
         </developer>
     </developers>
     <properties>
-        <jetty.version>9.2.11.v20150529</jetty.version>
+        <jetty.version>9.2.14.v20151106</jetty.version>
```

Comment:
I _think_ it _should_ be safe. The issue was that jetty 9.3 made java 8 a hard requirement.